dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -994,223 +1042,251 @@ protected Response saveContent(Contentlet contentlet, InitDataObject init) throw
+        if (UtilMethods.isSet(stInode)) {
+            ContentType type = APILocator.getContentTypeAPI(APILocator.systemUser()).find(stInode);
+            if (type != null && InodeUtils.isSet(type.inode())) {
                 // basic data
                 contentlet.setContentTypeId(type.inode());
-                if(map.containsKey("languageId")) {
-                    contentlet.setLanguageId(Long.parseLong((String)map.get("languageId")));
-                }
-                else {
-                    contentlet.setLanguageId(APILocator.getLanguageAPI().getDefaultLanguage().getId());
+                if (map.containsKey("languageId")) {
+                    contentlet.setLanguageId(Long.parseLong((String) map.get("languageId")));
+                } else {
+                    contentlet.setLanguageId(
+                            APILocator.getLanguageAPI().getDefaultLanguage().getId());
                 }
 
                 // check for existing identifier
-                if(map.containsKey("identifier")) {
+                if (map.containsKey("identifier")) {
                     contentlet.setIdentifier(String.valueOf(map.get("identifier")));
                     try {
-                        Contentlet existing=APILocator.getContentletAPI().findContentletByIdentifier((String)map.get("identifier"), false,
-                                contentlet.getLanguageId(), APILocator.getUserAPI().getSystemUser(), false);
+                        Contentlet existing = APILocator.getContentletAPI()
+                                .findContentletByIdentifier((String) map.get("identifier"), false,
+                                        contentlet.getLanguageId(),
+                                        APILocator.getUserAPI().getSystemUser(), false);
                         APILocator.getContentletAPI().copyProperties(contentlet, existing.getMap());
                         contentlet.setInode("");
                     } catch (Exception e) {
-                        Logger.debug(this.getClass(), "can't get existing content for ident "+map.get("identifier")+" lang "+contentlet.getLanguageId() + " - creating new one");
+                        Logger.debug(this.getClass(),
+                                "can't get existing content for ident " + map.get("identifier")
+                                        + " lang " + contentlet.getLanguageId()
+                                        + " - creating new one");
                     }
                 }
 
                 // build a field map for easy lookup
-                Map<String,Field> fieldMap=new HashMap<String,Field>();
-                for(Field ff : new LegacyFieldTransformer(type.fields()).asOldFieldList()){
+                Map<String, Field> fieldMap = new HashMap<String, Field>();
+                for (Field ff : new LegacyFieldTransformer(type.fields()).asOldFieldList()) {
                     fieldMap.put(ff.getVelocityVarName(), ff);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 175 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)